catalog-rest-service/src/main/java/org/openmetadata/catalog/exception/PipelineServiceVersionException.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.openmetadata.catalog.exception;
+
+import javax.ws.rs.core.Response;
+
+public class PipelineServiceVersionException extends WebServiceException {
+
+  private static final String BY_NAME_MESSAGE = "Pipeline Service Version mismatch due to [%s].";
+
+  public PipelineServiceVersionException(String message) {
+    super(Response.Status.BAD_REQUEST, message);
```

Comment:
nit: I think here we should return a 500 error since the error is not associated with the request parameters.